frappe/model/document.py

```diff
@@ -1092,7 +1092,9 @@ def run_post_save_methods(self):
 			self.run_method("on_update_after_submit")
 
 		self.clear_cache()
-		self.notify_update()
+
+		if not hasattr(self.flags, "notify_update") or self.flags.notify_update == True:
```

Comment:
```suggestion
		if not hasattr(self.flags, "notify_update") or self.flags.notify_update:
```

Reply:
Not `getattr(self.flags, "notify_update", False)`?

Reply:
@gavindsouza default is true, Also `getattr` for dict always returns something - `None`.